cmake.sh

```diff
@@ -1,14 +1,14 @@
 package: CMake
 version: "%(tag_basename)s"
-tag: "v3.16.3-alice2"
+tag: "v3.17.0"
 source: https://github.com/alisw/CMake
 build_requires:
  - "GCC-Toolchain:(?!osx)"
  - make
 prefer_system: "(?!osx)"
```

Comment:
If the bug is fixed in 3.17, then we can use system CMake if it is recent enough, right?

Reply:
Can you remove the `"(?!osx)"` part at `prefer_system`? If `3.17` fixes the bug we should use the brew version instead of always building our own.